simulator/src/ux.js

```diff
@@ -375,8 +375,17 @@ export function showProperties(obj) {
                     s = `<p><span>${prop.name}</span><input class='objectPropertyAttribute' type='number'  name='${prop.func}' min='${prop.min || 0}' max='${prop.max || 200}' value=${obj[attr]}></p>`;
                     $(moduleProperty.modulePropertyInner).append(s);
                 } else if (obj.mutableProperties[attr].type === 'text') {
-                    s = `<p><span>${prop.name}</span><input class='objectPropertyAttribute' type='text' autocomplete='off'  name='${prop.func}' maxlength='${prop.maxlength || 200}' value=${obj[attr]}></p>`;
+                    s = `<p><label for="color-picker">${prop.name}</label> <div class='color-picker'style='display: flex; align-items: center;'>
+                    <input class='objectPropertyAttribute' type='text'aria-labelledby="color-picker-label" style='width:1000%;' autocomplete='off' name='${prop.func}' maxlength='${prop.maxlength || 200}' value=${obj[attr]}>
```

Comment:
why it's `1000%` ?

Reply:
Bhaiya , i want this width to be maximum , as by hit and trial if a keep this width as 100% then it's spacing is not proper. we can also do this by applying the `min-width:91%`.

https://github.com/CircuitVerse/CircuitVerse/assets/96644946/1fd43865-a1d3-44e9-8f11-2cf97b3a29d1


should a apply min-width property instead?

i don’t know i should write this or not but..
Really Thank you bhaiya for analysis the code in such a way, I haven’t contribute yet in such a big organisation only do contributes in small projects of societies and clubs of college, Really sorry bhaiya, for facing problems in my code writing style.

Reply:
Adding arbitary value may work for some screen and browser but there will be chance to break for other case.

Reply:
You can do this, instead of using flex, use absolute positioning for color picker.
Read this blog - https://css-tricks.com/absolute-positioning-inside-relative-positioning/

Reply:
Done Bhaiya, i have updated css , instead of using flex i use absolute positioning for color-Picker
<img width="1680" alt="Screenshot 2024-03-24 at 7 10 29 PM" src="https://github.com/CircuitVerse/CircuitVerse/assets/96644946/28ea4234-345b-4cd4-ac80-71a0ff15be67">